packages/components/src/toggle-group-control/toggle-group-control/component.tsx

```diff
@@ -49,7 +48,7 @@ function ToggleGroupControl(
 	} = useContextSystem( props, 'ToggleGroupControl' );
 	const cx = useCx();
 	const containerRef = useRef();
-	const [ resizeListener, sizes ] = useResizeAware();
+	const [ resizeListener ] = useResizeAware();
```

Comment:
It looks like `react-resize-aware` was only needed to resize the backdrop, and so I think we can remove this line and the import at the top of this file (cc'ing @ntsekouras for a confirmation!)

Reply:
That makes sense actually. The listener isn't very useful if the `sizes` aren't being used. I removed it for now

Reply:
Yeah, it seems it can be safely removed.